src/index.js

```diff
@@ -15,7 +15,8 @@ function newInfoWindow(hotel) {
 }
 
 export default React.createClass({
-    propTypes: {hotels: React.PropTypes.array.isRequired},
+    propTypes: {hotels: React.PropTypes.array.isRequired,
+                mapMarkerImgUrl: React.PropTypes.string},
```

Comment:
Is `mapMarkerImgUrl` a good name? What do you think of `markerClusterImageUrlPrefix`?